src/lib/EEPROM/elrs_eeprom.cpp

```diff
@@ -18,21 +19,32 @@ ELRS_EEPROM::Begin()
 }
 
 uint8_t
-ELRS_EEPROM::ReadByte(const uint16_t address)
+ELRS_EEPROM::ReadByte(const unsigned long address)
 {
-    if (address >= RESERVED_EEPROM_SIZE) {
+    if (address >= RESERVED_EEPROM_SIZE)
+    {
         // address is out of bounds
+        Serial.println("ERROR! EEPROM address is out of bounds");
         return 0;
     }
     return EEPROM.read(address);
 }
 
 void
-ELRS_EEPROM::WriteByte(const uint16_t address, const uint8_t value)
+ELRS_EEPROM::WriteByte(const unsigned long address, const uint8_t value)
 {
-    if (address >= RESERVED_EEPROM_SIZE) {
+    if (address >= RESERVED_EEPROM_SIZE)
+    {
         // address is out of bounds
+        Serial.println("ERROR! EEPROM address is out of bounds");
         return;
     }
     EEPROM.write(address, value);
+
+#ifdef PLATFORM_ESP32
+    if (!EEPROM.commit())
```

Comment:
We probably don't want per-byte commit. Can we move the commits up to a transactional level so that there's just one per save?

Reply:
sweet